examples/chip-tool-darwin/templates/commands.zapt

```diff
@@ -31,6 +31,7 @@ public:
         {{#if isArray}}
         {{else if isStruct}}
         {{else if (isOctetString type)}}
+        AddArgument("{{asUpperCamelCase label}}", &{{>field}});
```

Comment:
If this is the same as the `isString`, case, why not just remove this case altogether, since isString is true for both octet strings and char strings?
```suggestion
```

Reply:
Done. Thank you.